pkgs/applications/emulators/snes9x/default.nix

```diff
@@ -29,40 +34,37 @@ stdenv.mkDerivation rec {
       "snes9x-gtk"
     else
       "snes9x";
-  version = "1.61";
+  version = "1.62";
 
   src = fetchFromGitHub {
     owner = "snes9xgit";
     repo = "snes9x";
     rev = version;
     fetchSubmodules = true;
-    sha256 = "1kay7aj30x0vn8rkylspdycydrzsc0aidjbs0dd238hr5hid723b";
+    hash = "sha256-RcxFNmUbJp0rUugWOqQa3Sy/Hh18ZPOeDTxC0JY5GJQ=";
   };
 
   patches = [
     # Fix cross-compilation, otherwise it fails to detect host compiler features
     # Doesn't affect non CC builds
     (fetchpatch {
-      url = "https://mirror.its.dal.ca/gentoo-portage/games-emulation/snes9x/files/snes9x-1.53-cross-compile.patch";
-      sha256 = "sha256-ZCmnprimz8PtDIXkB1dYD0oura9icW81yKvJ4coKaDg=";
+      url = "https://github.com/snes9xgit/snes9x/commit/f39ab408f4151c16d44e45470cc0736ffb2803f8.patch";
+      hash = "sha256-GMlHBsADEF+rycmEVgpWy220hZwld5D2e8fsYA7HblM=";
     })
   ];
```

Comment:
Can be removed since we are not using `autoreconfHook` anymore.

Reply:
This is a patch I contributed upstream which includes new `configure` file which we don't need to rebuild and which will probably be includes with Snes9x 1.63. The old patch doesn't work due to some refactors in `configure.ac` which make it not merge cleanly.